android/sentry.properties

```diff
@@ -0,0 +1,4 @@
+defaults.url=https://sentry.io/
+defaults.org=hathor
+defaults.project=wallet-mobile
+auth.token=1d9fa34e90034945a983323f9f7a1b9039b00d6a507d4246b02658c79d0c0cb8
```

Comment:
I agree that it should be kept in a config file. Can we use envvars?

Reply:
Even though it will be packed with the final AAB. I'm not sure if anyone can download and inspect an AAB. Do you know, @obiyankenobi and @jansegre ?

Reply:
I have no idea. But we need to check exactly what they are. @pedroferreira1 was telling me that sentry recommends adding it to source control, so they might not need to be secret.

Reply:
This is a file generated automatically by sentry when configuring it in react native. Sentry recommends adding it to the version control, that's why I commited it and didn't change it.

https://docs.sentry.io/platforms/react-native/

> The Sentry Wizard will guide you through the process of setting everything up correctly. This has to be done only once, and the files created can go into your version control system.

I am still studying more to understand exactly if there is any harm in having this exposed here.

Reply:
So I've found a stackoverflow reply of one of the Sentry developers saying the opposite of the Sentry official docs. https://stackoverflow.com/a/48376918/2117670

> We recommend treating a sentry.properties like an .env file.

It is basically the same, so you should add it to your e.g. .gitignore.

The reason why it's called sentry.properties is because of android gradle, we needed it to be read natively.

Reply:
Done

Reply:
Maybe we should commit a template file without `auth.token`. I think the rest of the information is not sensitive. And explain in the README that we need this file to be created in the `ios` and `android` folders. Or just add the file template directly on the README.

Reply:
Added to README. Can you see if it's good?

Reply:
Good!